runners/s3-benchrunner-c/CMakeLists.txt

```
@@ -23,6 +23,14 @@ include(FetchContent)
 FetchContent_Declare(json URL https://github.com/nlohmann/json/releases/download/v3.11.2/json.tar.xz)
 FetchContent_MakeAvailable(json)
 
+# CLI parser
+FetchContent_Declare(
+    argh_parser
+    GIT_REPOSITORY https://github.com/adishavit/argh
+    GIT_TAG v1.3.2
+)
```

Seems simpler/faster to just download the code, vs having git involved.
This is what we're doing with json (above)
```suggestion
FetchContent_Declare(argh_parser URL https://github.com/adishavit/argh/archive/refs/tags/v1.3.2.tar.gz)
```